From d93e4223f00e25df1fe1c539eb328370dc7ba473 Mon Sep 17 00:00:00 2001
From: Raviteja Narayanam <raviteja.narayanam@xilinx.com>
Date: Mon, 9 Nov 2020 15:45:07 +0530
Subject: [PATCH 1751/1852] regulator: da9121: Add details for da9121

commit 221c796d5ebd0b43bc323b58fa2ee9bf052025bc from
https://github.com/Xilinx/linux-xlnx.git

Added the devices and their details as supported by DA9121 driver.

Signed-off-by: Raviteja Narayanam <raviteja.narayanam@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/regulator/Kconfig | 21 ++++++++++++++-------
 1 file changed, 14 insertions(+), 7 deletions(-)

diff --git a/drivers/regulator/Kconfig b/drivers/regulator/Kconfig
index 3385143f1bc9..a0e9d1b6831f 100644
--- a/drivers/regulator/Kconfig
+++ b/drivers/regulator/Kconfig
@@ -267,13 +267,20 @@ config REGULATOR_DA9063
 	  will be called da9063-regulator.
 
 config REGULATOR_DA9121
-        tristate "Dialog Semiconductor DA9121 regulator"
-        depends on I2C
-        select REGMAP_I2C
-        help
-          Say y here to support for the Dialog Semiconductor DA9121.
-          The DA9121 is a single-channel dual-phase buck converter.
-          Controlled through an I2C interface.
+	tristate "Dialog Semiconductor DA9121/DA9130/DA9217/DA9122/DA9131/DA9220/DA9132 regulator"
+	depends on I2C
+	select REGMAP_I2C
+	help
+	  Say y here to support for the Dialog Semiconductor DA9121
+	  family of devices.
+
+	  The DA9121/DA9130/DA9217/DA9122/DA9131/DA9220/DA9132 regulators are
+	  controlled through an I2C interface. This driver can also be built
+	  as a module.
+	  DA9121/DA9130 - Single channel dual phase 10A buck converter
+	  DA9217	- Single channel dual phase 6A buck converter
+	  DA9122/DA9131 - Dual channel single phase  5A buck converter
+	  DA9220/DA9132 - Dual channel single phase  3A buck converter
 
 config REGULATOR_DA9210
 	tristate "Dialog Semiconductor DA9210 regulator"
-- 
2.31.1

